src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -34,60 +34,6 @@ const CONTROL_KEYS = {
     S: 83,
 };
 
-// TODO: can be optimize for some uses
-var presiceSlerp = function presiceSlerp(qb, t) {
```

Comment:
Typo (all over) :It's 'precise' not presice

Reply:
done